BUILD.md

```diff
@@ -1412,3 +1416,155 @@ HTTP/1.1 200 OK
 ```
 
 </details>
+
+### Dynamically Overwrite build properties
+
+Dependending on where are in your development process, you might want to dynamically overwrite a build property. One scenario is for example to set the `mainBuildBranch` option which is used also to clone collections and initialize the DBB collections. See [Perform Impact Build for topic branches](#perform-impact-build-for-topic-branches)
+
+To dynamically overwrite any build property, you can make use of the `--propOverwrite` cli argument. To overwrite a build property, pass it in as a key-value pair via the CLI. Several key-value pairs can be passed by comma-separating them :`--propOverwrite mainBuildBranch=develop,cobol_compilerVersion=V6`. Please note, that values which contain a comma (`,`) cannot be passed into the list. Please also make sure that you don't run into any limitations with regards of the length of the overall build command. For mass overwrites, use the `--propFiles` option to pass in one or several property files.
+
```

Comment:
We have some case of value with coma. It is not possible to allow escaping such values ?
`buildOrder=BMS.groovy\,Cobol.groovy`

Reply:
Hi Jean-Yves, I have performed multiple tests, but it did not work out - unfortunately ...